Documentation/ceph-cluster-crd.md

```diff
@@ -30,6 +30,13 @@ metadata:
   name: rook-ceph
   namespace: rook-ceph
 spec:
+# enabling network requires NetworkAttachmentDefinition `rook-public-nw` and `rook-cluster-nw` to be present in the `rook-ceph` namespace.
```

Comment:
These high level examples really just focus on the minimal settings, with no commented settings. I still think the multus example does not need to be included in these high level examples.

Reply:
removed the commented settings